lms/values.yaml

```diff
@@ -36,6 +36,7 @@ lms:
   replicaCount: 3
   releaseName: "molt-lms"
   sourceDialect: ""
+  targetDialect: ""
```

Comment:
What do you think about setting this default to be "cockroach" still that way it's at least filled in for most users.

Reply:
Good idea, done.